libraries/chain/include/eosio/chain/unapplied_transaction_queue.hpp

```diff
@@ -242,10 +243,15 @@ class unapplied_transaction_queue {
    }
 
    template<typename Itr>
-   void removed( Itr itr ) {
+   void removed( Itr itr, bool call_next = true ) {
       if( itr->trx_type == trx_enum_type::incoming || itr->trx_type == trx_enum_type::incoming_persisted ) {
          --incoming_count;
       }
+      if( call_next && itr->next ) {
+         transaction_trace_ptr trace = std::make_shared<transaction_trace>();
+         trace->id = itr->trx_meta->id();
```

Comment:
Yeah, I agree. I switched this to indicate duplicate transaction which is what it would have generated before this was integrated into the unapplied_transaction_queue.